Makefile

```diff
@@ -24,6 +24,9 @@ help:  # Display this help.
 dev: # Open the package in xcode
 	xed .
 
+test: # Run the app unit tests
+	@xcodebuild test -project GravatarApp.xcodeproj -scheme GravatarApp -destination 'platform=iOS Simulator,name=iPhone 16 Pro'
```

Comment:
Notice this now uses iPhone 16 Pro. This is a bit of an arbitrary change, but I noticed that if I run the tests on my machine with iPhone 16, the Simulator fails to launch:

```
Test target AnalyticsTests encountered an error (Failed to clone device named 'iPhone 16'. (Underlying Error: The operation couldn’t be completed. Device was allocated but was stuck in creation state.  Check CoreSimulator.log for more information.))
```

I tried to run the tests via Fastlane's `scan` and it automatically picked up iPhone 16 Pro and succeeded to run the tests.

When I changed the command to use iPhone 16 Pro on my end via vanilla `xcodebuild` like CI does, it worked.